src/gp-model/config.py

```diff
@@ -1,19 +1,27 @@
 import os
+
+import urllib3
 from pymongo import MongoClient
 from dotenv import load_dotenv
 from pathlib import Path
 BASE_DIR = Path(__file__).resolve().parent
 dotenv_path = os.path.join(BASE_DIR, '.env')
 load_dotenv(dotenv_path)
 
+urllib3.disable_warnings()
+
 
 class Config:
     DEBUG = False
     TESTING = False
     CSRF_ENABLED = True
     SECRET_KEY = os.getenv("SECRET_KEY")
     CREDENTIALS = os.getenv('GOOGLE_APPLICATION_CREDENTIALS')
-
+    MEASUREMENTS_TOPIC = os.getenv('MEASUREMENTS_TOPIC')
+    AIRQLOUDS_TOPIC = os.getenv('AIRQLOUDS_TOPIC')
+    BOOTSTRAP_SERVERS = os.getenv('BOOTSTRAP_SERVERS')
+    GP_MODEL_DB = os.getenv('GP_MODEL_DB')
+    GP_MODEL_DB_URI = os.getenv('GP_MODEL_DB_URI')
```

Comment:
Secret manager needs to be updated @uman95

Reply:
Could you update those of prod @NoahNsimbe 
@uman95 may only have access to staging

Reply:
Curious about the information required in AirQlouds.
When an AirQloud is just created, it does not have any Sites in it until is "refreshed".

I am curious as to how this subscription to AirQlouds topic will be utilised. @uman95 / @NoahNsimbe 

Thanks!

Reply:
Thanks @Baalmart, the coordinates and sites under the AirQlouds are the most important fields required. As to newly created AirQlouds not having sites, we don't immediately build a spat-temp model (and consequently heatmap) on AirQlouds not even on newly added AirQlouds. It's more or less a request before building such model, which I believe such AirQloud must have been functional with available sites.

Reply:
Thanks @Baalmart for highlighting this. Let me review the implementation.